### PR TITLE
Add modal support

### PR DESCRIPTION
**Pull request by jnnnthnn**

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -82,97 +90,151 @@ class BerbixVerify extends React.Component {
...
         scrolling="no"
         referrerPolicy="no-referrer-when-downgrade"
       />
     );
+    if (this.props.showInModal) {
+      const disp = Math.max(this.state.height, window.innerHeight);
+      const marginTop = disp > 500 ? (disp - 500) * 0.25 : 10;
+      return (
+        <div
+          className="--berbix-overlay"
```
Comment (ericlevine) [inline src/index.jsx]: This is unused, can be removed.

File: src/index.jsx [hunk 2]
```diff
@@ -82,97 +90,151 @@ class BerbixVerify extends React.Component {
...
+          height: this.state.height + "px",
+          overflow: "hidden"
         }}
         allow="camera"
         scrolling="no"
         referrerPolicy="no-referrer-when-downgrade"
       />
     );
+    if (this.props.showInModal) {
+      const disp = Math.max(this.state.height, window.innerHeight);
```
Comment (ericlevine) [inline src/index.jsx]: Not sure if I follow the logic on this one - the intention is to get the height of the entire screen, whereas `this.state.height` is the height of the iframe.

File: src/index.jsx [hunk 3]
```diff
@@ -82,97 +90,151 @@ class BerbixVerify extends React.Component {
...
+            right: 0,
+            bottom: 0,
+            backgroundColor: "rgba(0, 0, 0, 0.6)",
+            zIndex: 1000
+          }}
+        >
+          <div
+            style={{
+              margin: "0 auto",
+              top: "100px",
```
Comment (ericlevine) [inline src/index.jsx]: I believe this can be omitted (and from the vanilla version as well).

File: src/index.jsx [hunk 4]
```diff
@@ -82,97 +90,151 @@ class BerbixVerify extends React.Component {
...
+              width: "100%",
+              maxWidth: "500px",
+              maxHeight: "100%",
+              overflow: "auto",
+              borderRadius: "5px",
```
Comment (ericlevine) [inline src/index.jsx]: Is this intentional?

Comment (jnnnthnn) [reply src/index.jsx]: Hm, nice catch. Leftover from a previous attempt.

**Pull request merged**